lib/virt_autotest/common.pm

```diff
@@ -244,6 +264,16 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             location => 'http://mirror.suse.cz/install/SLP/SLE-12-SP5-Server-LATEST/x86_64/DVD1/',
             linuxrc => 'ifcfg="eth0=192.168.122.113/24,192.168.122.1,192.168.122.1"',
         },
+        sles15sp4 => {
+            name => 'sles15sp4',
+            autoyast => 'autoyast_xen/sles11sp4PVx32_PRG.xml',
+            extra_params => '--os-variant sle15-unknown',    # problems after kernel upgrade
```

Comment:
'--os-variant' was replaced with '--osinfo' in SLE15+. try '--osinfo' may avoid some problems.

Reply:
no, '--osinfo' was newly introduced. I don't think it is supported in SLE12SP3, but '--os-variant' will be obselete quite soon either.

Reply:
'--os-variant'  issue can't be addressed here.  There are other staff which need to improve in common.pm too. We will address '--os-variant'  issue together with them in another ticket.

Reply:
It is okay to address in other ticket. Can you share the ticket link?

Reply:
Not yet opened. just some rough ideas. not in todo list of this roadmap.

Reply:
poo opened, https://progress.opensuse.org/issues/111806